tidalapi/request.py

```diff
@@ -34,7 +34,7 @@
 
 import requests
 
-from tidalapi.exceptions import ObjectNotFound, TooManyRequests
+from tidalapi.session import from_http_error
```

Comment:
Why are we importing from session instead of exceptions when from_http_error is defined there?